### PR TITLE
Add instructions for restoring IntelliJ run configurations #6818

### LINKED ISSUE #6818: Add instructions for restoring IntelliJ run configurations
**Problem**: IntelliJ run configurations are removed by IntelliJ upon importing the project.

**Solution**: Add instructions to restore run configurations using git after importing the project.

### PR DESCRIPTION
**Pull request by nashmia-riaz**
Fixes #6818 

Added instructions to the file docs/settingUp.md

The following lines were added after line 135
"
1. During your import, the run configurations for the project were deleted locally. In order to restore these, open git shell and run the command
  ```sh
     git checkout -- .idea/runConfigurations
  ```
"

### CODE REVIEW

File: docs/settingUp.md [hunk 1]
```diff
@@ -133,6 +133,10 @@ More information can be found at [this documentation](https://help.github.com/ar
    * Click `Next`.
    * Check `Use auto-import`. Ensure that `1.7` is used for the `Gradle JVM`.
    * Click `Finish`.
+1. During your import, the run configurations for the project were deleted locally. In order to restore these, open git shell and run the command
```
Comment (samsontmr) [inline docs/settingUp.md]: "The project's run configurations may have been deleted locally when you imported the project. To restore them, ..."

Another way of restoring would be to discard the changes caused by the deletion. If your method does not create any additional changes in git, then it is acceptable.

Comment (nashmia-riaz) [inline docs/settingUp.md]: Do you mean by resetting the branch i.e. by using the "git reset" command?

Comment (samsontmr) [inline docs/settingUp.md]: @nashmia-riaz Yes that's what I meant. However on second thought, these files are untracked so that might not work..

**Comment by samsontmr** [comment]
@wkurniawan07 Does your PR make this totally obsolete? If so, we can close this and give @nashmia-riaz credit for his contribution.

**Comment by wkurniawan07** [comment]
Indeed it is. @nashmia-riaz the issue has been resolved in another way that actually targeted the root cause instead of resolving the symptoms, but still thanks for your contribution!

**Pull request closed without merging**